app/src/components/molecules/HeaderButton.tsx

```diff
@@ -1,23 +1,34 @@
 import React from 'react';
 import { Button } from '@mui/material';
+import Cookies from 'universal-cookie';
 
-interface HeaderButtonProps {
+interface IHeaderButtonProps {
   children?: React.ReactNode;
   label: string;
   url: string;
+  icon?: any;
 }
 
-function HeaderButton(props: HeaderButtonProps) {
+function HeaderButton(props: IHeaderButtonProps) {
+  const cookies = new Cookies();
+
   function navTo(pageName: string) {
     return () => {
-      window.location.href = `${pageName}`;
+      if (pageName === '/logout') {
+        cookies.remove('user');

```

Comment:
Not sure if there's a better lcoation for this logic, it works but might be better suited to a helper file for all cookie/login stuff.

Reply:
Good enough for a first draft 👍